external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -373,16 +377,20 @@ private static Options buildOptions () {
         curatorFramework.start();
         String partitionPrefix = "partition_";
         String zkPath = oldKafkaSpoutOffsetQuery.getZkPath();
-        if (!zkPath.endsWith("/")) {
-            zkPath += "/";
+         if (zkPath.endsWith("/")) {
```

Comment:
Is there a reason for removing trailing / here and then appending in the loop below?  The loop already has a check for curatorFramework.checkExists().forPath(path). May be we can just print and exit therE? Also, can you call the printUsageAndExit method here to be consistent? Or is there a reason you could not use it?